meson.build

```diff
@@ -35,9 +35,15 @@ if get_option('buildtype').startswith('debug')
   pgrn_c_args += ['-DPGROONGA_DEBUG=1']
 endif
 
+# Note: Meson's split() is NOT shell-compatible
+# It splits only on whitespace and doesn't understand shell quoting
+# For example: 'a "b c" d' becomes ['a', '"b', 'c"', 'd'] instead of ['a', 'b c', 'd']
+# This works for most PostgreSQL flags but may break with quoted paths containing spaces
+pg_cppflags = run_command(pg_config, '--cppflags', check: true).stdout().strip().split()
+pg_cflags_sl = run_command(pg_config, '--cflags_sl', check: true).stdout().strip().split()
+
 postgresql = declare_dependency(
-  compile_args:
-    run_command(pg_config, '--cflags_sl', check: true).stdout().strip(),
+  compile_args: pg_cppflags + pg_cflags_sl,
```

Comment:
Could you give me an idea to improve these lines?
The current implementation isn't shell-compatible now.
So if we can, we want to fix it here.

Reply:
Meson has a feature to split shell command line internally: https://github.com/mesonbuild/meson/blob/f9e2ac805f99fd168cb1120220281ab0b1717f16/mesonbuild/utils/universal.py#L1160-L1230

But it's not exposed. Could you open an issue on https://github.com/mesonbuild/meson/ and ask the best way for this situation?

Reply:
Sure. Thank you for your handling the issue too.